mmv1/third_party/terraform/resources/resource_storage_bucket.go.erb

```diff
@@ -343,6 +344,13 @@ func resourceStorageBucket() *schema.Resource {
 				Computed:    true,
 				Description: `Enables uniform bucket-level access on a bucket.`,
 			},
+<% unless version == "ga" -%>
+			"public_access_prevention": {
+				Type:        schema.TypeString,
+				Optional:    true,
```

Comment:
Most likely will need
`Computed: true` here

Reply:
Computed means that the API can/will return this field even if it is not set.

Reply:
This is so if the field is returned in the instance that terraform does not explicitly set it. Terraform will not blow up. 

We can see how the tests go to determine if they fail or not first so we can see what will be needed.

Reply:
Ah, so it's based on the behavior of the API itself? I guess I need to see what it returns for buckets where this is not configured. Presumably, if we see nothing as a result then I'll want to omit the computed option, but if I see something set to false, I'll want to include it. Let me have a nose around.

Reply:
Your understanding is correct